pkg/apis/kubeless/v1beta1/zz_generated.deepcopy.go

```diff
@@ -86,16 +85,15 @@ func (in *FunctionList) DeepCopy() *FunctionList {
 func (in *FunctionList) DeepCopyObject() runtime.Object {
 	if c := in.DeepCopy(); c != nil {
 		return c
-	} else {
-		return nil
 	}
+	return nil
```

Comment:
may be you can use it as return since existing code is not returning nil

Reply:
@rakesh-garimella Actually this code gets generated by `code-generator` and here I've updated it to `kubernetes-1.8.7. Additionally, this file is not meant to be manually updated.